lardataobj/RecoBase/Hit.cxx

```diff
@@ -115,8 +115,8 @@ namespace recob {
       return a.Channel() < b.Channel();
     if (a.View() != b.View())
       return a.View() < b.View();
-    if (a.StartTick() != b.StartTick())
-      return a.StartTick() < b.StartTick();
+    if (a.PeakTime() != b.PeakTime())
```

Comment:
I think PeakTime is a double?
I think this comparison will very often fail.  Is that the intent here?

Reply:
Actually PeakTime is a float.  The intent is that two hits will almost certainly not have the same PeakTime unless they are the same hit.  This was not the case with the previous comparison using StartTick since distinct but closely spaced hits that were merged in hit candidate finding can have the same StartTick and therefore cannot be distnguished.